### PR TITLE
feat: bump dependencies

### PR DESCRIPTION
**Pull request by smira**
| Package | Update | Change |
|---|---|---|
| git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git | major | `20221109` -> `20221214` | 
| [containerd/containerd](https://togithub.com/containerd/containerd) | patch | `v1.6.12` -> `v1.6.14` | 
| git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git | patch | `5.15.82` -> `6.1.1` | 
| [https://git.tukaani.org/xz.git](https://git.tukaani.org/xz) | minor | `v5.2.9` -> `5.4.0` |
| https://github.com/ipxe/ipxe.git | digest | `563bff4` -> `d879c8e` |
| NDIVIA | major | `515.65.01` -> `525.60.13` |

DRBD disabled until they have a version compatible with Linux 6.1, or we could use in-tree DRBD (not sure which version is that ??)

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

**Comment by smira** [comment]
drbd 9.2.1 build seems to be broken:

```


#84 8.414 /tmp/build/0/drbd/drbd_state.c: In function 'twopc_retry_timeout':
--
45765 | #84 8.417 /tmp/build/0/drbd/drbd_state.c:4504:27: error: implicit declaration of function 'prandom_u32'; did you mean 'prandom_u32_max'? [-Werror=implicit-function-declaration]
45766 | #84 8.417  4504 \|                 timeout = prandom_u32() % timeout;
45767 | #84 8.417       \|                           ^~~~~~~~~~~
45768 | #84 8.417       \|                           prandom_u32_max
45769 | #84 8.484 /tmp/build/0/drbd/drbd_transport_tcp.c: In function 'dtt_wait_for_connect':
45770 | #84 8.484 /tmp/build/0/drbd/drbd_transport_tcp.c:642:19: error: implicit declaration of function 'prandom_u32'; did you mean 'prandom_u32_max'? [-Werror=implicit-function-declaration]
45771 | #84 8.484   642 \|         timeo += (prandom_u32() & 1) ? timeo / 7 : -timeo / 7; /* 28.5% random jitter */
45772 | #84 8.484       \|                   ^~~~~~~~~~~
45773 | #84 8.484       \|                   prandom_u32_max
45774 | #84 9.368 cc1: some warnings being treated as errors
```

File: kernel/prepare/pkg.yaml [hunk 1]
```diff
@@ -42,9 +42,6 @@ steps:
         make mrproper
       - |
         cd /toolchain && git clone https://github.com/a13xp0p0v/kconfig-hardened-check.git
-      - |
-        patch -p1 < /pkg/patches/hardening.kconfig.patch
```
Comment (frezbo) [inline kernel/prepare/pkg.yaml:46]: no more hardening patches needed?

Comment (smira) [reply kernel/prepare/pkg.yaml:46]: this was called confusingly, but it patched Kconfig to make hardening checks pass :-D so it's actually better now than it was

**Comment by smira** [comment]
I disabled drbd build for now :(

**Comment by smira** [comment]
https://pub.nethence.com/storage/drbd-source-troubles

File: Pkgfile [hunk 2]
```diff
@@ -39,7 +39,7 @@ vars:
   # renovate: datasource=github-releases depName=flannel-io/cni-plugin
   flannel_cni_version: v1.2.0
   flannel_cni_ref: 18a3027e7d03feeb6ecdfdbc3bf254a8c8b38b04
```
Comment (frezbo) [inline Pkgfile:41]: ref needs updating

**Review by frezbo — APPROVED** [review]

File: Pkgfile (hunk 2 above)
Comment (smira) [reply Pkgfile:41]: ah, there was no sha update, it was just extra space which got removed it looks like

**Pull request merged**